src/popup/modules/UserInterface.js

```diff
@@ -109,8 +109,18 @@ const refreshQrCode = throttle(() => {
         CommonMessages.hideError();
     }
 
-    QrCreator.setTextInternal(text);
-    QrCreator.generate();
+    try {
+        QrCreator.setTextInternal(text);
+        QrCreator.generate();
+    } catch (e) {
+        // Error thrown from qrcodegen & kjua wrapper when code too long
+        if (e === "Data too long") {
+            CommonMessages.showError("errorQrCodeOverflow", true);
```

Comment:
Why should it be dismissable? IMHO, it should not.

The error is automatically hidden, when the user inputs valid text later, AFAIK…

Reply:
Means i throw the error regardless? Is that what you meant?

Reply:
No, not exactly. I meant the second parameter of the function [`showError`](https://tinywebex.github.io/MessageHandler/module-CommonMessages.html#.showError).